resources/scripts/components/server/ServerConsole.tsx

```diff
@@ -121,7 +124,7 @@ export default () => {
                             className={'mr-1'}
                         />
                         &nbsp;{bytesToHuman(disk)}
-                        <span className={'text-neutral-500'}> / {server.limits.disk} MB</span>
```

Comment:
any reason you removed the styling here?

Reply:
I originally moved it to the if statement in the first pr...... Ahh I see you..... Yeh I'll fix it when I get home.

Reply:
I'm going to have to play with it after work, as we don't need "MB" at the end of Unlimited, one of the other reasons I moved it to the IF statement.

Reply:
Fixed.